bindata/etcd/pod.yaml

```diff
@@ -54,56 +54,20 @@ spec:
                            --endpoints=${ALL_ETCD_ENDPOINTS}"
         ${ETCDCTL} member list
 
-        echo "waiting for member ${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}..."
-        COUNT=30
-        while [ $COUNT -gt 0 ]; do
-          echo "current member list is..."
-          ${ETCDCTL} member list
-          echo ""
-          echo ""
+        export ETCD_INITIAL_CLUSTER=$(discover-etcd-initial-cluster \
```

Comment:
@sttts when `discover-etcd-intial-cluster` returns non-zero, I'd like to print it's output and exit